extensions/amp-ad/0.1/amp-ad-custom.js

```diff
@@ -136,7 +136,7 @@ export class AmpAdCustom extends AMP.BaseElement {
 
       // Get the parent body of this amp-ad element. It could be the body of
       // the main document, or it could be an enclosing iframe.
-      const body = ancestorElementsByTag(this.element, 'BODY')[0];
+      const body = closestByTag(this.element, 'BODY');
```

Comment:
Hmm. This doesn't look right. Though I guess not the subject to your PR. Would you mind leaving a `TODO` here for me to investigate?